test/contracts/front/challenges.js

```diff
@@ -388,6 +394,27 @@ describe('Challenges Contract', () => {
       expect(await getChallengeSequenceNumber(challengeId)).to.equal('0');
     });
 
+    it('Can resolve if last resolved challenge is 0', async () => {
+      await setLastChallengeResolvedSequenceNumber(resolver, 0);
+      await injectChallengeWithSequenceNumber(cooldown - 1);
+      expect(await canResolve(resolver, challengeId)).to.be.true;
+      await expect(resolveChallenge(challengeId, resolver)).to.be.eventually.fulfilled;
+    });
+
+    it('Can resolve if exactly cooldown is between last resolved challenge and new one', async () => {
```

Comment:
`exactly` suggests it needs to be exactly one challenge. `at least` would be better.